BouncyCastle/src/AESFastEngine.crysl

```diff
@@ -0,0 +1,13 @@
+SPEC org.bouncycastle.crypto.engines.AESFastEngine
+
+OBJECTS
+	int dummy;
```

Comment:
why did you name this 'dummy'?

Reply:
I wanted to use the same naming as in the [AESEngine rule](https://github.com/CROSSINGTUD/Crypto-API-Rules/blob/master/BouncyCastle/src/AESEngine.crysl). It is added because there needs to be an object in the OBJECTS section or else in eclipse there is an error marked.